engine/src/main/resources/application.properties

```diff
@@ -4,7 +4,6 @@
 quarkus.kafka.devservices.port=9092
 
 quarkus.http.port=8087
-
```

Comment:
Empty lines be like
![image](https://github.com/RedHatInsights/notifications-backend/assets/3845764/2488a1ad-32cf-4e0d-80fa-b2fd352ab69c)